backend/api/v1/mail_templates/testmail/hahaha/weekly_for_dba_example.html

```diff
@@ -0,0 +1,168 @@
+                white-space: pre-wrap;
+                border: 1px solid #ddd;
+                padding: 8px;
+            }
+
+            #slow-query-table tr:nth-child(even){background-color: #f2f2f2;}
+
+            #slow-query-table tr:hover {background-color: #ddd;}
+
+            #slow-query-table th {
+                padding-top: 12px;
+                padding-bottom: 12px;
+                text-align: left;
+                background-color: #4f46e5;
+                color: white;
+            }
+        </style>
+    </head>
+    <body style="background-color: #ffffff;font-family:-apple-system,BlinkMacSystemFont,&quot;Segoe UI&quot;,Roboto,Oxygen-Sans,Ubuntu,Cantarell,&quot;Helvetica Neue&quot;,sans-serif">
+        <h1 data-id="mail-heading" class="mail-heading">Database slow query weekly report {{BEGIN_DATE}} ~ {{END_DATE}}</h1>
+        <p data-id="mail-first-graph" class="mail-text text-center">
+            Below is the slow query for {{BEGIN_DATE}} ~ {{END_DATE}}, each database lists at most 5 slow queries.
+            You can visit&nbsp;<a href="{{VISIT_URL}}"><span class="mail-text">https://bytebase.com</span></a>&nbsp;to enable/disable slow query collection.
+            <br>
+        </p>
+        <!-- TODO(junyi): For loop this div block to generate multiple environments block.-->
+        <div style="margin-top: 32px;">
+            <h2 class="env-title text-center"> Prod </h2>
+            <!-- TODO(junyi): For loop this div block to generate multiple instance in one environment.-->
+            <div style="margin-top: 24px;">
+                <h2 class="instance-tile text-center"><a href="{{INSTANCE_LINK}}">Instance 1</a></h2>
+                <table id="slow-query-table" style="margin:auto;">
+                    <tr>
+                        <th>DB</th>
+                        <th>Slow Query</th>
+                    </tr>
+                    <!-- TODO(junyi) For loop this tr block to generate all slow query entry in table element.-->
+                    <tr>
+                        <td>Employee</td>
+                        <td> -- create a dummy table with 10 rows
+CREATE TABLE dummy_table (id INT NOT NULL PRIMARY KEY);
+INSERT INTO dummy_table VALUES (1), (2), (3), (4), (5), (6), (7), (8), (9), (10);
+-- create the table if it doesn't exist
+CREATE TABLE my_table (
+    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    name VARCHAR(50),
+    age INT,
+    salary INT
+);
+
+-- insert 1,000,000 rows into the table
+INSERT INTO my_table (name, age, salary)
+SELECT CONCAT('Name', t1.id), FLOOR(RAND() * 47) + 18, FLOOR(RAND() * 40000) + 10000
+FROM dummy_table t1, dummy_table t2, dummy_table t3, dummy_table t4, dummy_table t5, dummy_table t6, dummy_table t7, dummy_table t8;
+
+-- run a complex query that takes more than 10 seconds to execute
+SELECT t1.name, AVG(t1.salary)
+FROM my_table t1
+JOIN my_table t2 ON t1.id = t2.id
+WHERE t1.age > 30
+GROUP BY t1.name
+HAVING COUNT(*) > 1000;
+                        </td>
```

Comment:
prefer using monospace font for codes wdyt?

Reply:
Done, using monospace font for sql-text class.